airbyte-cdk/bulk/toolkits/extract-jdbc/src/main/kotlin/io/airbyte/cdk/read/JdbcPartitionReader.kt

```diff
@@ -57,6 +59,15 @@ sealed class JdbcPartitionReader<P : JdbcPartition<*>>(
     override fun releaseResources() {
         acquiredResources.getAndSet(null)?.close()
     }
+
+    /** If configured max feed read time elapsed we exit with a transient error */
+    protected fun checkMaxReadTimeElapsed() {
+        sharedState.configuration.maxSnapshotReadTime?.let {
+            if (java.time.Duration.between(sharedState.readStartTime, Instant.now()) > it) {
+                throw TransientErrorException("Shutting down reader: max time elapsed")
```

Comment:
The error message should have the word "snapshot" in there at least. Also replace "time" with "duration" please.

Reply:
Changed to "Shutting down snapshot reader: max duration elapsed"